### PR TITLE
Fix post-> deposit error

### LINKED ISSUE #560: Unable to Create Solution Post
**Describe the bug**
When attempting to post a solution or idea to the devhub portal I get the below error.

**To Reproduce**
1. Go to 'https://near.social/devhub.near/widget/app?page=create'
2. Click on 'Submit'
3. Approve the Transaction

**Expected behavior**
Solution posted to site successfully

**Screenshots**
image

**Desktop:**
 - OS: [e.g. iOS] Windows
 - Browser [e.g. chrome, safari] Chrome and Brave
 - Version [e.g. 22] 1.61.101 Chromium 1200.6099.71

**Additional context**
Tried with multiple different wallet providers.  Not sure what is unique about my account that is causing the issue.

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Resolves #560 
Preview [link](https://near.org/megha19.near/widget/app?page=feed) 
For testing, please use an account which hasn't granted devgogigs account permission (for notifications)

I have tested: (using ref from [here](https://github.com/near/neardevhub-widgets/pull/552/files))
- add_post
- edit_post
- reply -> comment, idea, attestation, solution, sponsorship
- add_like

Let me know if there is anything missing

### CODE REVIEW

**Comment by frol** [comment]
It used to be 0.1 NEAR deposited with that call to cover the cost of storing the permission data. 0.1 NEAR is too much, but the current 10e5 yoctoNEAR might be too little if it is a fresh account and it had not deposited any tokens to cover storage on social.near contract. @Megha-Dev-19 Let's go with 0.01 NEAR

File: src/devhub/entity/post/PostEditor.jsx [hunk 1]
```diff
@@ -226,6 +226,7 @@ const onSubmit = () => {
           keys: [context.accountId + "/index/notify"],
         },
         gas: Big(10).pow(14),
+        deposit: Big(10).pow(5),
```
Comment (frol) [inline src/devhub/entity/post/PostEditor.jsx:229]:
```suggestion
        deposit: Big(10).pow(22),
```

File: src/devhub/page/create.jsx [hunk 2]
```diff
@@ -191,6 +191,7 @@ const onSubmit = () => {
           keys: [context.accountId + "/index/notify"],
         },
         gas: Big(10).pow(14),
+        deposit: Big(10).pow(5),
```
Comment (frol) [inline src/devhub/page/create.jsx:194]:
```suggestion
        deposit: Big(10).pow(22),
```

**Comment by Megha-Dev-19** [comment]
@frol I tested with 0.1 NEAR and 10e5 yoctoNEAR and it works with both
but sure will update it

**Review by frol — APPROVED** [review]

**Comment by frol** [comment]
@Megha-Dev-19 I have updated it from the UI, so let's wait for CI and merge it

**Pull request merged**